web/react/components/view_image.jsx

```diff
@@ -5,113 +5,142 @@ var Client = require('../utils/client.jsx');
             var self = this;
             var img = new Image();
             img.load(this.getPreviewImagePath(filename),
-                function(){
+                function() {
                     var progress = self.state.progress;
                     progress[id] = img.completedPercentage;
-                    self.setState({ progress: progress });
-            });
+                    self.setState({progress: progress});
+                });
             img.onload = function(imgid) {
                 return function() {
                     var loaded = self.state.loaded;
                     loaded[imgid] = true;
-                    self.setState({ loaded: loaded });
-                    $(self.refs.image.getDOMNode()).css("max-height",imgHeight);
+                    self.setState({loaded: loaded});
+                    $(self.refs.image.getDOMNode()).css('max-height', imgHeight);
                 };
             }(id);
             var images = this.state.images;
             images[id] = img;
-            this.setState({ images: images });
+            this.setState({images: images});
         } else {
             // there's nothing to load for non-image files
             var loaded = this.state.loaded;
             loaded[id] = true;
-            this.setState({ loaded: loaded });
+            this.setState({loaded: loaded});
         }
     },
     componentDidUpdate: function() {
         if (this.state.loaded[this.state.imgId]) {
             if (this.refs.imageWrap) {
-                $(this.refs.imageWrap.getDOMNode()).removeClass("default");
+                $(this.refs.imageWrap.getDOMNode()).removeClass('default');
             }
         }
     },
     componentDidMount: function() {
         var self = this;
-        $("#"+this.props.modalId).on('shown.bs.modal', function() {
-            self.setState({ viewed: true });
+        $('#' + this.props.modalId).on('shown.bs.modal', function() {
+            self.setState({viewed: true});
             self.loadImage(self.state.imgId);
-        })
+        });
 
-        $(this.refs.modal.getDOMNode()).click(function(e){
-            if (e.target == this || e.target == self.refs.imageBody.getDOMNode()) {
+        $(this.refs.modal.getDOMNode()).click(function(e) {
+            if (e.target === this || e.target === self.refs.imageBody.getDOMNode()) {
                 $('.image_modal').modal('hide');
             }
         });
 
         $(this.refs.imageWrap.getDOMNode()).hover(
             function() {
-                $(self.refs.imageFooter.getDOMNode()).addClass("footer--show");
+                $(self.refs.imageFooter.getDOMNode()).addClass('footer--show');
             }, function() {
-                $(self.refs.imageFooter.getDOMNode()).removeClass("footer--show");
+                $(self.refs.imageFooter.getDOMNode()).removeClass('footer--show');
             }
         );
 
+        $(this.getDOMNode()).on('keyup', function(e) {
```

Comment:
Thanks for the input. Right now our code base follows the convention of using JQuery for these types of event bindings, and this is consistent with that practice.